Integrations/src/main/java/io/deephaven/integrations/python/PythonDeephavenSession.java

```diff
@@ -56,9 +56,7 @@ public class PythonDeephavenSession extends AbstractScriptSession<PythonSnapshot
     private static final Logger log = LoggerFactory.getLogger(PythonDeephavenSession.class);
 
     private static final String DEFAULT_SCRIPT_PATH = Configuration.getInstance()
-            .getProperty("PythonDeephavenSession.defaultScriptPath")
-            .replace("<devroot>", Configuration.getInstance().getDevRootPath())
-            .replace("<workspace>", Configuration.getInstance().getWorkspacePath());
+            .getStringWithDefault("PythonDeephavenSession.defaultScriptPath", ".");
```

Comment:
Related to this change:
`io.deephaven.engine.util.ScriptFinder#findScriptEx(java.lang.String, java.lang.String)` has some comments that probably need cleanup after devroot is retired.

Reply:
Ha! Just mentioned this in the previous comment before I noticed this.

Reply:
https://github.com/deephaven/deephaven-core/issues/2978